packaging/linux/deb/package_binaries.sh

```diff
@@ -81,6 +81,14 @@ build_one_architecture() {
     > "$cron_file"
   chmod 755 "$cron_file"
 
+  # Whitelist for NativeMessaging
+  kbnm_bin="/usr/bin/kbnm"
+  kbnm_file="$dest/etc/opt/chrome/native-messaging-hosts/io.keybase.kbnm"
+  mkdir -p "$(dirname "$kbnm_file")"
```

Comment:
You continue to be the only person I've ever met who quotes expressions like this properly :-D